package.json

```diff
@@ -20,7 +20,6 @@
     "@grafana/runtime": "9.4.7",
     "@grafana/ui": "9.4.7",
     "lodash": "4.17.21",
-    "qs": "6.11.1",
```

Comment:
Package was being used for the LS Notebook search params, updated to use the LS browser native URL API with `URLSearchParam`